syfertext/span.py

```diff
@@ -289,6 +302,7 @@ def create_pointer(
         if owner is None:
             owner = span.owner
 
+        """ Gives unexpected keyword argument """
```

Comment:
can you give more details here?

Reply:
I did not get this error while running any tests, but my linter threw this error
Here is the definition of StatePointer, the init does not have the attribute type garbage_collect_data,
Might/Might not be an issue
```
class StatePointer(ObjectPointer):
    """This class defines a pointer to a State object. Whenever a State
    object is searched on the grid and found on any of the remote workers,
    a pointer to it, represented by this object, is returned. Then using
    this object, a copy of the state can be downloaded to the worker that
    requested it.
    """

    def __init__(
        self,
        location: BaseWorker,
        id_at_location: str,
        owner: BaseWorker,
    ):
        # Initialize the parent object
        super(StatePointer, self).__init__(
            location=location,
            id_at_location=id_at_location,
            owner=owner,
            id=None,
            garbage_collect_data=False,
        )
```